<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnwrapKeyDlg</class>
 <widget class="QDialog" name="UnwrapKeyDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Unwrap key</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="7" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Mechanism</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Slot status</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Slot ID</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="mSlotIDText"/>
       </item>
       <item row="3" column="3">
        <widget class="QLineEdit" name="mSessionText"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Slot list</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Label</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Session</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Wrapped Key</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
       <item row="3" column="5">
        <widget class="QLineEdit" name="mLoginText"/>
       </item>
       <item row="4" column="5">
        <widget class="QPushButton" name="mFindBtn">
         <property name="text">
          <string>Find</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="6">
        <widget class="QComboBox" name="mSlotsCombo"/>
       </item>
       <item row="4" column="1" colspan="4">
        <widget class="QLineEdit" name="mWrapKeyPathText"/>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Unwrap Key properties</string>
         </property>
        </widget>
       </item>
       <item row="6" column="5">
        <widget class="QLineEdit" name="mUnwrapObjectText"/>
       </item>
       <item row="6" column="4">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Object</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="3">
        <widget class="QComboBox" name="mUnwrapLabelCombo"/>
       </item>
       <item row="7" column="1">
        <widget class="QComboBox" name="mUnwrapMechCombo"/>
       </item>
       <item row="7" column="2">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Param</string>
         </property>
        </widget>
       </item>
       <item row="7" column="3" colspan="3">
        <widget class="QLineEdit" name="mUnwrapParamText"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Key</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="4" column="0">
        <widget class="QCheckBox" name="mPrivateCheck">
         <property name="text">
          <string>Private</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="mUnwrapCheck">
         <property name="text">
          <string>Unwrap</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QComboBox" name="mTypeCombo"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Class</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="mModifiableCheck">
         <property name="text">
          <string>Modifiable</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Object</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QLineEdit" name="mObjectText"/>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="mClassCombo"/>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Attributes</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QCheckBox" name="mTokenCheck">
         <property name="text">
          <string>Token</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Key Properties</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Label</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="mPrivateCombo"/>
       </item>
       <item row="4" column="2">
        <widget class="QCheckBox" name="mSensitiveCheck">
         <property name="text">
          <string>Sensitive</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QComboBox" name="mSensitiveCombo"/>
       </item>
       <item row="4" column="4">
        <widget class="QCheckBox" name="mWrapCheck">
         <property name="text">
          <string>Wrap</string>
         </property>
        </widget>
       </item>
       <item row="4" column="5">
        <widget class="QComboBox" name="mWrapCombo"/>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="mUnwrapCombo"/>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="mModifiableCombo"/>
       </item>
       <item row="7" column="1">
        <widget class="QComboBox" name="mTokenCombo"/>
       </item>
       <item row="5" column="3">
        <widget class="QComboBox" name="mEncryptCombo"/>
       </item>
       <item row="6" column="3">
        <widget class="QComboBox" name="mSignCombo"/>
       </item>
       <item row="5" column="5">
        <widget class="QComboBox" name="mDecryptCombo"/>
       </item>
       <item row="6" column="5">
        <widget class="QComboBox" name="mVerifyCombo"/>
       </item>
       <item row="5" column="2">
        <widget class="QCheckBox" name="mEncryptCheck">
         <property name="text">
          <string>Encrypt</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QCheckBox" name="mSignCheck">
         <property name="text">
          <string>Sign</string>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="QCheckBox" name="mDecryptCheck">
         <property name="text">
          <string>Decrypt</string>
         </property>
        </widget>
       </item>
       <item row="6" column="4">
        <widget class="QCheckBox" name="mVerifyCheck">
         <property name="text">
          <string>Verify</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="5">
        <widget class="QLineEdit" name="mLabelText"/>
       </item>
       <item row="7" column="3">
        <widget class="QComboBox" name="mExtractableCombo"/>
       </item>
       <item row="7" column="2">
        <widget class="QCheckBox" name="mExtractableCheck">
         <property name="text">
          <string>Extractable</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UnwrapKeyDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UnwrapKeyDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
